src/libs/ajax/loaded-data/useLoadedData.ts

```diff
@@ -31,7 +44,8 @@ export type UseLoadedDataResult<T> = [
  * }
  * @returns a tuple with [currentLoadedState, updateDataMethod]
  */
-export const useLoadedData = <T>(): UseLoadedDataResult<T> => {
+export const useLoadedData = <T>(hookArgs?: UseLoadedDataArgs<T>): UseLoadedDataResult<T> => {
+  const args: UseLoadedDataArgs<T> = hookArgs ? hookArgs : {}
```

Comment:
I think `||` is more common in terra-ui for this concept
```suggestion
  const args: UseLoadedDataArgs<T> = hookArgs || {}
```

Reply:
i like it.  Let me get this in for final commit.

Reply:
scratch that... looks like this would cripple unit test branch coverage and falsely report 100% branch coverage even if the true and false paths are not both being tested.  I will add this as caution/avoid to the code guidance...

Reply:
Could it instead be replaced with a `?.` later on in the code?

Reply:
removed by switching over to default function arg value.